src/shims/manual/Directory.Build.props

```diff
@@ -22,7 +22,9 @@
     <ContractAssemblyPath>$(NetFxRefPath)$(MSBuildProjectName).dll</ContractAssemblyPath>
     <DefineConstants Condition="'$(TargetsNetCoreApp)' == 'true'">$(DefineConstants);netcoreapp</DefineConstants>
     <!-- Use runtime assemblies for the TargetGroup, using build configuration OSGroup -->
-    <RefPath>$(ArtifactsBinDir)runtime/$(TargetGroup)-$(_bc_OSGroup)-$(_bc_ConfigurationGroup)-$(ArchGroup)/</RefPath>
+	<_runtimeOSGroup>$(_bc_OSGroup)</_runtimeOSGroup>
```

Comment:
Nit: indentation in this and the next kind is off

Reply:
line

Reply:
bad editor! (new machine n++ using tabs, embarrassing)